launch/launch/substitutions/environment_variable.py

```diff
@@ -30,23 +30,32 @@ class EnvironmentVariable(Substitution):
     If the environment variable is not found, it returns empty string.
     """
 
-    def __init__(self, name: SomeSubstitutionsType) -> None:
+    def __init__(self, name: SomeSubstitutionsType, *,
+                 default_value: SomeSubstitutionsType = '') -> None:
```

Comment:
Please wrap all the arguments or none of them, also, never indent more than one time between two lines. Also for multi line function/method declarations, always end the closing `)` on its own line at the same indention level as the opening `(`.

You may have multiple arguments per line, like this:

```python
    def __init__(
        self, name: SomeSubstitutionsType, *,
        default_value: SomeSubstitutionsType = ''
    ) -> None:
```

But I would recommend doing one per line unless you have an unreasonable number of arguments with short names, e.g. `xx, xy, xz, yx, yy, yz, zx, zy, zz` or similar:

```python
    def __init__(
        self,
        name: SomeSubstitutionsType,
        *,
        default_value: SomeSubstitutionsType = ''
    ) -> None:
```

For example as to why, in the above it's easy to see that `default_value` is a required keyword because the `*,` is more visible, whereas it's easily overlooked in the current code and in the previous example.